### DISCUSSION TITLE
Could we have a simpler and readable version of the ontology overview diagram

### DISCUSSION THREAD

**Discussion by kheerand** [ternaustralia/ontology_tern#71] [General] [answered]
Hi @edmondchuc,

I'm wondering if you have a simpler version of [this diagram](https://swift.rc.nectar.org.au/v1/AUTH_05bca33fce34447ba7033b9305947f11/static-web-assets/linkeddata-website/latest/tern-ontology/tern-ontology-overview.png)? One that just has the class name and the relationship between them without all the details within each class?  A bit hard to read this one on a screen at full res.  

As discussed via email, the fonts generated from TB Composer is too small to be able to read easily in a document or a presentation.  So a version that could be used for explaining or discussing with folks would be really useful to have.

Thanks
Kheeran

**Comment by smguru** [comment]
@kheerand We may need to write a separate class diagram, @edmondchuc  was doing that, We will update the class diagram

**Comment by edmondchuc** [comment, marked as answer]
@kheerand here's a copy of the new diagram without class attributes. 
![tern-ontology-conceptual-simple](https://user-images.githubusercontent.com/37032744/153308374-49690fa7-71c5-41c4-a892-47aacb964bd0.png)

**Reply by kheerand** [reply]
Yup, this is much more readable and clearer to understand and explain to someone.  Please include in the documentation when you get a chance.

**Comment by edmondchuc** [comment]
If this diagram is useful, I will add it to the documentation when I'm back from leave.

**Reply by miekeGR** [reply]
Very nice @edmondchuc I am finding it useful.
